components/cards/cards.css

```diff
@@ -0,0 +1,164 @@
+h2,h4,p {
+    margin:0;
+    padding:0;
+}
+.image{
+    height:100%;
+    width:100%;
+    object-fit:cover;
+}
+.card-container{
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap:wrap;
+    height:100vh;
+}
+.card{
+    flex-basis:25rem;
+    min-width:21.875rem;
+    background-color:var(--cards-background-color);
+    border-radius:0.313rem;
+    overflow:hidden;
+    margin:1.25rem;
+    border:1px solid var(--dark-color);
+}
+
+
+.button{
+    border:none;
+    color:var(--button-color);
+    text-transform:uppercase;
+    background-color:transparent;
+    font-size:medium;
+    font-weight:bold;
+    cursor:pointer;
+}
+i{
+    color:var(--icons-color);
+    font-size:1.25rem;
+    cursor:pointer;
+}
+h4{
```

Comment:
just a suggestion , you can give classes intead of applying style directly on an element

Reply:
sure,I'll do it right away